conda_build/build.py

```diff
@@ -107,6 +107,10 @@ def have_prefix_files(files):
             # skip symbolic links (as we can on Linux)
             continue
 
+        # dont try to mmap an empty file
+        if os.stat(path).st_size == 0:
```

Comment:
I'm not sure.  I don't know what your CTAGS file is there.  Does it make sense to lstat first, then stat with an or clause?  Replacing stat with lstat seems unsafe to me (just turn off following symlinks?  It seems like some (most?) of the time, we would want to follow them?